src/tests/GfxReplayTest.cpp

```diff
@@ -357,6 +357,20 @@ TEST(GfxReplayTest, simulatorIntegration) {
   simConfig.enableGfxReplaySave = true;
 
   auto sim = Simulator::create_unique(simConfig);
+  auto objAttrMgr = sim->getObjectAttributesManager();
+  objAttrMgr->loadAllJSONConfigsFromPath(
+      Cr::Utility::Directory::join(TEST_ASSETS, "objects/nested_box"), true);
+
+  auto handles = objAttrMgr->getObjectHandlesBySubstring("nested_box");
```

Comment:
Related to the JSON changes: I'm modifying this test to add a render object instance and verify its pose gets serialized/deserialized with reasonable precision.